docs/user-documentation/content_types.md

```diff
@@ -0,0 +1,126 @@
+
+!!! tip This page will address how to create and modify ingest forms (or rather, content types) via the GUI. For help working with forms via the API, please check out the [Further Reading](#further-reading) section for links to more advanced Drupal documentation.
+
+Since objects in Islandora 8 are stored in Drupal as Nodes, we use the standard Drupal Content Types system to create and edit our ‘ingest forms’ [Content Types in Drupal 8](https://www.drupal.org/docs/8/administering-drupal-8-site/managing-content-0/working-with-content-types-and-fields). Islandora 8 forms are Drupal forms, and if you are already familiar with Drupal Field UI, you’re already well equipped to create and modify your own ingest forms in Islandora 8.
+
+This page will address how to create and modify ingest forms (or rather, content types) via the GUI. For help working with forms via the API, please check out the Further Reading section for links to more advanced Drupal documentation.
+
+## Modify a Content Type
+
+If you have deployed your Islandora 8 with the Islandora Demo configuration, you will already have a Repository Item content type available, with pre-configured fields. In the Admin menu, go to Structure >> Content Types and find the Repository Item content type. Select *Manage Fields*.
+
+![a screenshot of the Add Content Type page](/docs/assets/islandora8_managefields.png)
+
+You will see a list of the fields that are already available in this content type. 
+
+### Add a Field
+
+Let’s add a new field where a user can indicate if the repository item needs to be reviewed:
+
+1. Click “Add Field”
+1. Since this is a “yes/no” decision, we will use “Boolean” and give the field a name. [List of Drupal 8 FieldTypes, FieldWidgets, and FieldFormatters](https://www.drupal.org/docs/8/api/entity-api/fieldtypes-fieldwidgets-and-fieldformatters)
+1. Now we configure how the field is stored in the Drupal database. For this field type we can select how many values will be allowed. Set this value to "1":
+1. Now we configure how the field is described (including its display label and the help text for when it appears on a form) and constraints on its use. In this case, the field will be required for this Content Type, and will be set to “on” by default: 
+
+![a screenshot of the field settings page](/docs/assets/islandora8_fieldsettings.png)
+
+*Save*. Now we have added our new field: 
+
+![a screenshot of a "Needs Review?" field in the Drupal field UI](/docs/assets/islandora8_newfield.png)
+
+And it appears in the ingest form when we try to create a new repository object:
+
+![a screenshot of a "Needs Review?" field appearing at the bottom of a new node form](/docs/assets/islandora8_newfieldinform.png)
+
+!!! tip New fields, with the exception of Typed Relation fields, are not automatically indexed in Fedora and the triple-store. Update the Content Type's RDF Mapping [link that section of the docs] to enable indexing the field. 
```

Comment:
Since we have the RDF Mapping as part of this page, we should replace "[link that section of the docs]" with "(see below)".